backend/jest.config.js

```diff
@@ -1,4 +0,0 @@
-module.exports = {
-  preset: 'ts-jest',
-  testEnvironment: 'node',
-};
```

Comment:
How come delete this from BE?

Reply:
I assumed you guys wouldn't need jest to test but I can add it back in if needed!

Reply:
Oh wait but if we remove this will we still be able to run tests in BE?

Reply:
You won't be able to if you guys were intending to use ts-jest! Have you been using it for any so far?

Reply:
Hmm wait do you mind explaining which parts of the `GooglePlaceService` were ts-jest when it was in the BE?

Reply:
The old tests for GooglePlacesService were!

Reply:
I think we need to keep this for BE because we use it here https://github.com/ubclaunchpad/foodies/blob/master/backend/jest.config.js. We mainly need it because of this: https://github.com/ubclaunchpad/foodies/pull/76/files

Reply:
All good! I'll add it back in :)